README.md

```diff
@@ -475,6 +475,38 @@ On successful, it will restart the system and you should following logs on the c
     $root@OpenWrt:/#
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+##Using opkg utility
+All the packages built for pistachio-marduk has been hosted on [IMGCreator downloads server](https://downloads.imgcreator.io/pistachio/marduk), so it should be possible to 
+use opkg utility to install/upgrade/remove the OpenWrt packages.
+
+1. Check if you have /etc/opkg/distfeeds.conf pointing to [IMGCreator downloads server](https://downloads.imgcreator.io/pistachio/marduk), else for your local development you can edit this to point to your local webserver which has the required packages.
+
+            root@OpenWrt:/# sed -i "s|http://downloads.imgcreator.io/pistachio/marduk/|http://192.168.91.79:8080/|" /etc/opkg/distfeeds.conf
```

Comment:
Is your local server address |http://192.168.91.79:8080/| added by mistake here?